### PR TITLE
PIPE-5946 - add yamls

### PR DESCRIPTION
**Pull request by benharosh**

### CODE REVIEW

File: tests/yaml/S_DockerPush_5946_001.yml [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    configuration:
+      environmentVariables:
+        readOnly:
+          JFROG_CLI_BUILD_NUMBER: ${run_id}
+    steps:
+      - name: S_DockerPush_5946_001_1
+        type: DockerBuild
+        configuration:
+          affinityGroup: S_DockerPush_5946_001
+          dockerImageName: ${artifactory_registry_url}/test-automation-docker-local/s_dockerpush_5946
```
Comment (neha-mishraa) [inline tests/yaml/S_DockerPush_5946_001.yml:29]: can we have the camel-casing in image name(to have uniformity in naming convention used) or is it supposed to be in small letters?

Comment (benharosh) [reply tests/yaml/S_DockerPush_5946_001.yml:29]: small letters should be fine. That's what we are using for all our pipeline docker images, e.g. pipelines-api:0.0.0-m000

File: tests/yaml/S_DockerPush_5946_001.yml [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+          dockerImageTag: ${run_id}
+          inputResources:
+            - name: S_DockerPush_5946_001_GitRepo
+          integrations:
+            - name: s_artifactory
+        execution:
+          onStart:
+            - artifactory_registry_url=$(echo "$int_s_artifactory_url" | sed 's#.*//\([^/]*\)/.*#\1#')
```
Comment (neha-mishraa) [inline tests/yaml/S_DockerPush_5946_001.yml:37]: can you add a comment on what is being done here?

File: tests/yaml/S_DockerPush_5946_001.yml [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+      - name: S_DockerPush_5946_001_3
+        type: Bash
+        configuration:
+          inputSteps:
+            - name: S_DockerPush_5946_001_2
+          integrations:
+            - name: s_artifactory
+        execution:
+          onExecute:
+            - fileStepArtifacts=$(jfrog rt search "*/dockerpush_5946/${run_id}/*" | jq -r '.[] | "\(.path) \(.props.sha256 | first )"')
```
[the remark this answers is not in the document]
Comment (benharosh) [reply tests/yaml/S_DockerPush_5946_001.yml:65]: I added a comment in the yml file

File: tests/yaml/S_DockerPush_5946_001.yml (hunk 2 above)
Comment (benharosh) [reply tests/yaml/S_DockerPush_5946_001.yml]: I added a comment in the yml file

**Pull request merged**